lib/srv/desktop/rdp/rdpclient/build.rs

```diff
@@ -14,11 +14,21 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+use std::time::SystemTime;
+
 fn main() {
+    let timestamp = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .unwrap()
+        .as_secs();
+
+    let autogen_warning = format!("// Code generated by cbindgen at {timestamp:?}. DO NOT EDIT.");
```

Comment:
We're formatting a number, we don't need the debug formatting
```suggestion
    let autogen_warning = format!("// Code generated by cbindgen at {timestamp}. DO NOT EDIT.");
```